tslib/src/functions-throttled/sync-results-to-sql/sync-results.data.service.ts

```diff
@@ -93,12 +96,235 @@ export class SyncResultsDataService implements ISyncResultsDataService {
+                                            browserMinorVersion,
+                                            browserPatchVersion,
+                                            uaOperatingSystemLookup_id,
+                                            uaOperatingSystemMajorVersion,
+                                            uaOperatingSystemMinorVersion,
+                                            uaOperatingSystemPatchVersion,
+                                            navigatorPlatformLookup_id,
+                                            navigatorLanguageLookup_id,
+                                            navigatorCookieEnabled,
+                                            networkConnectionDownlink,
+                                            networkConnectionEffectiveTypeLookup_id,
+                                            networkConnectionRtt,
+                                            screenWidth,
+                                            screenHeight,
+                                            outerWidth,
+                                            outerHeight,
+                                            innerWidth,
+                                            innerHeight,
+                                            colourDepth,
+                                            deviceOrientationLookup_id,
+                                            appUsageCount,
+                                            userAgentLookup_id,
+                                            ident)
+        VALUES (@batteryIsCharging,
+                @batteryLevelPercent,
+                @batteryChargingTimeSecs,
+                @batteryDischargingTimeSecs,
+                @cpuHardwareConcurrency,
+                @browserFamily_lookup_id,
+                @browserMajorVersion,
+                @browserMinorVersion,
+                @browserPatchVersion,
+                @uaOperatingSystemLookup_id,
+                @uaOperatingSystemMajorVersion,
+                @uaOperatingSystemMinorVersion,
+                @uaOperatingSystemPatchVersion,
+                @navigatorPlatformLookup_id,
+                @navigatorLanguageLookup_id,
+                @cookieEnabled,
+                @networkConnectionDownlink,
+                @networkConnectionEffectiveTypeLookup_id,
+                @networkConnectionRoundTripTimeMs,
+                @screenWidth,
+                @screenHeight,
+                @outerWidth,
+                @outerHeight,
+                @innerWidth,
+                @innerHeight,
+                @colourDepth,
+                @deviceOrientationLookup_id,
+                @appUsageCount,
+                @userAgentLookup_id,
+                @ident);
+
+        SET @userDeviceId = (SELECT SCOPE_IDENTITY());
+
+        UPDATE mtc_results.checkResult
+           SET userDevice_id = @userDeviceId
+         WHERE id = @checkResultId;
+    `
+    // tslint:enable:no-trailing-whitespace
```

Comment:
please also review this directive when you pull in the updated linter

Reply:
The new linter is in!